test/java/seedu/addressbook/parser/UtilsTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.parser;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.common.Utils;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*; 
+
+public class UtilsTest {
+
+	@Test
+	public void isAnyNullTest() {
+			Utils Util = new Utils();
+		    Name name = null;
+			
+			//test case 1: return true for a name that is actually a null
+			assertTrue(Utils.isAnyNull(name));
+	}
+	
+	@Test
+	public void isAnyNullTest1() {
+		try {
+			Utils Util = new Utils();
+			Name name2 = new Name("Andy Tan");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.